### PR TITLE
Fix: ttlSecondsAfterFinished not taking effect.

### PR DESCRIPTION
**Pull request by Syulin7**
if tfjob completed, it would not reconcile again, ttlSecondsAfterFinished won't take effect even if configured.

### CODE REVIEW

File: pkg/controller.v1/tensorflow/job.go [hunk 1]
```diff
@@ -225,21 +225,28 @@ func (tc *TFController) cleanupTFJob(tfJob *tfv1.TFJob) error {
 		return nil
 	}
 	duration := time.Second * time.Duration(*ttl)
-	if currentTime.After(tfJob.Status.CompletionTime.Add(duration)) {
+	if tfJob.Status.CompletionTime == nil {
```
Comment (cheyang) [inline pkg/controller.v1/tensorflow/job.go:228]: Why is this change necessary?

Comment (Syulin7) [reply pkg/controller.v1/tensorflow/job.go:228]: Optimized the performance of the operator to avoid unnecessary reconciles of completed tfjob.

**Review by cheyang — APPROVED** [review]
